runtimevar/gcpruntimeconfig/example_test.go

```diff
@@ -76,11 +76,59 @@ func ExampleOpenVariable() {
 	_ = cfg
 }
 
+func Example_openVariableHowto() {
```

Comment:
Why not just fix the existing example? I'd rather not have "Howto"-specific examples.

Reply:
In this case, because the existing example is doing too much. It's also calling `Latest`, which I don't want to show here.

Perhaps we could enhance the example gathering script to snip off everything from some point on? We kinda have it with `// Ignore unused variables in example` now, but this is confusing when people are just looking at the example.

In general, using exactly a single source for examples and howtos is challenging because users reach this code in two ways - (1) reading the howto and (2) looking at the example code in godoc.org

I could perhaps rename this, if having "howto" in it doesn't seem good?

Reply:
Let's just drop the `Latest` from the existing example then, and maybe replace it with a comment like `// ... use the runtimevar.Variable`, or nothing.